src/SessionManager/ProgramSettings.h

```diff
@@ -98,6 +101,19 @@ struct ProgramSettings {
         warning_msgs.clear();
         debug_msgs.clear();
     }
+
+    std::string ParseBrowserCommand(const std::string& url) {
+        const std::string wildcard = "%url";
```

Comment:
We can't use `%url`; this will be interpreted by FreeDesktop as `%u` in a desktop file on Linux. I would prefer something simpler like `CARTA_URL`.